src/Framework/Models/Model.php

```diff
@@ -245,11 +249,47 @@ protected function validatePropertyType($key, $value)
      * @since 2.19.6
      *
      * @param $key
+     *
      * @return string
      */
     protected function getPropertyType($key)
     {
-        return strtolower(trim($this->properties[$key]));
+        $type = is_array($this->properties[$key]) ? $this->properties[$key][0] : $this->properties[$key];
+
+        return strtolower(trim($type));
+    }
+
+    /**
+     * Get the default for a property if one is provided, otherwise default to null
+     *
+     * @unreleased
+     *
+     * @param $key
+     *
+     * @return mixed|null
+     */
+    protected function getPropertyDefault($key)
+    {
+        return is_array(
```

Comment:
This combination of a multi-line, multiple logic check combined with an inline ternary can be a bit confusing. This could be resolved with changing to linebreaks.

```php
return is_array(...) && isset(...)
    ? $this->properties[$key][$i]
    " null;
```

Reply:
Fixed in f419cfccc